cobc/error.c

```diff
@@ -57,13 +58,36 @@ static void
 print_error_prefix (const char *file, int line, const char *prefix)
 {
 	if (file) {
+		char *absfile = NULL ;
+		if (cb_diagnostics_absolute_paths
+		    && strcmp (file, COB_DASH) != 0
+		    && file[0] != '/'
+		    && file[0] != '\\'
+		    && ( file[0] != 0 && file[1] != ':' )
+			){
+			int filelen = strlen (file);
+			int dirlen = 256;
+			char *cwd ;
+			absfile = cobc_malloc( dirlen + 1 + filelen + 1 );
+			cwd = getcwd (absfile, dirlen);
```

Comment:
`getcwd()` needs a define handling for WIN32, feel free to include that from libcob/fileio.c

Reply:
ok